testing/consumer-test/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "lint": "eslint . --ext .js,.ts",
-    "test": "ts-node run.ts"
+    "test": "ts-node run.ts --verbose"
```

Comment:
Do we want to keep this?

Reply:
I left it there because it is helpful to know when the pipeline fails, rather than just showing Failed : [4.5] and no error.